### PR TITLE
Separated awstpr and clustertpr

### LINKED ISSUE #24: Use separated awstpr and clustertpr
https://github.com/giantswarm/awstpr and https://github.com/giantswarm/clustertpr are now separate proejcts, we need to adjust aws-operator to use it.

### PR DESCRIPTION
**Pull request by nhlfr**
Fixes #24

### CODE REVIEW

**Comment by nhlfr** [comment]
This is the smaller PR I was talking about. Let's put #22 on hold until that one gets merged.

File: glide.yaml [hunk 1]
```diff
@@ -23,8 +23,9 @@ import:
 - package: k8s.io/client-go
   version: v2.0.0-alpha.1
 - package: github.com/giantswarm/awstpr
-  repo: git@github.com:giantswarm/awstpr.git
-  version: 3d98b40a2bc399a6a20f3b19c634d56d99a69e81
+  version: 99da9c8f8583749d0fef89d14b3cfa420e13ebc1
+- package: github.com/giantswarm/clustertpr
+  version: 16f7b5230a6850c1b7b62bad4f6002607f8ad3c2
```
Comment (xh3b4sd) [inline glide.yaml]: When `awstpr` is pinned there should be no reason to pin the cluster TPR itself. This repo should be pinned inside the `awstpr` repo.

File: main.go [hunk 2]
```diff
@@ -144,7 +142,6 @@ func main() {
 
 	daemonCommand.PersistentFlags().StringVar(&Flags.Aws.AccessKey.ID, "aws.accesskey.id", "", "ID of the AWS access key")
 	daemonCommand.PersistentFlags().StringVar(&Flags.Aws.AccessKey.Secret, "aws.accesskey.secret", "", "Secret of the AWS access key")
-	daemonCommand.PersistentFlags().StringVar(&Flags.Aws.Region, "aws.region", "", "Region in EC2 service")
```
Comment (xh3b4sd) [inline main.go]: I feel like I miss something. Why do we remove the AWS region now?

Comment (iaguis) [inline main.go]: We [included it in awstpr](https://github.com/giantswarm/awstpr/blob/master/aws/aws.go#L6)

File: glide.yaml (hunk 1 above)
Comment (nhlfr) [inline glide.yaml]: done

File: main.go (hunk 2 above)
Comment (nhlfr) [inline main.go]: As @iaguis said.
Also, we moved the client creation inside the code of "create" service.

**Pull request merged**